lib/log/main.go

```diff
@@ -17,6 +24,10 @@ var Log *zap.SugaredLogger
 // safe, but has a clunkier API). See: https://godoc.org/go.uber.org/zap#hdr-Choosing_a_Logger
 var RawLog *zap.Logger
 
+type Client = sentry.Client
```

Comment:
nit: I feel like this alias is more confusing, since there's already so many things called Client, and `sentry.Client` is more descriptive

Reply:
Good point -- this was copied from the tulip/sentry library, but I agree, I'll remove the alias